Connector/httputils/router.py

```diff
@@ -0,0 +1,293 @@
+                "message": f"Can not add {network} network for {coin} because it is already added"
+            }
+
+        coinHandler = currenciesHandler[coin]
+        ok, err = coinHandler.addConfig(network=network, config=config)
+        if not ok:
+            return {
+                "success": False,
+                "message": err
+            }
+
+        if coin not in self._availableCoins:
+            self._availableCoins[coin] = {
+                network: None
+            }
+        else:
+            self._availableCoins[coin][network] = None
+
+        return {
+            "success": True,
+            "message": f"{network} network added for currency {coin}"
+        }
+
+    async def removeCoin(self, coin, network):
+
+        ok, err = self.checkCoinNetworkIntegrity(coin=coin, network=network)
+        if not ok:
+            return {
+                "success": False,
+                "message": err
+            }
+
+        try:
+            self.checkIsAvailableRoute(coin=coin, network=network)
+        except error.NotFoundError:
+            logger.printError(f"{network} network has not been previously added for currency {coin}")
+            return {
+                "success": False,
+                "message": f"{network} network has not been previously added for currency {coin}"
+            }
+
+        logger.printInfo(f"Removing {network} network for currency {coin}")
+
+        del self._availableCoins[coin][network]
+        if len(self._availableCoins[coin]) == 0:
+            del self._availableCoins[coin]
+
+        coinHandler = currenciesHandler[coin]
+        ok, err = await coinHandler.removeConfig(network)
+
+        return {
+            "success": ok,
+            "message": f"{network} network removed for currency {coin}" if ok else err
+        }
+
+    def getCoin(self, coin, network):
+
+        ok, err = self.checkCoinNetworkIntegrity(coin=coin, network=network)
+        if not ok:
+            return {
+                "success": False,
+                "message": err
+            }
+
+        try:
+            self.checkIsAvailableRoute(coin=coin, network=network)
+        except error.NotFoundError:
+            logger.printError(f"{network} network has not been previously added for currency {coin}")
+            return {
+                "success": False,
+                "message": f"{network} network has not been previously added for currency {coin}"
+            }
+
+        logger.printInfo(f"Returning configuration for {network} network for currency {coin}")
+
+        coinHandler = currenciesHandler[coin]
+        config, err = coinHandler.getConfig(network)
+        if err is not None:
+            return {
+                "success": False,
+                "message": err
+            }
+
+        return {
+            "success": True,
+            "message": f"Config configuration for {network} network for currency {coin} retrieved successfully",
```

Comment:
"Config configuration for ..." redundant

Reply:
Replaced